BigBaseV2/src/backend/looped/weapons/infinite_mag.cpp

```diff
@@ -5,8 +5,6 @@ namespace big
 {
 	void looped::weapons_infinite_mag()
 	{
-		if (g->weapons.infinite_mag) {
-			WEAPON::SET_PED_INFINITE_AMMO_CLIP(PLAYER::PLAYER_PED_ID(), g->weapons.infinite_mag);
-		}
```

Comment:
Forcefully running this native each tick even when disabled might cause side effects.